src/externalServices/storeTrigger/requestCall.ts

```diff
@@ -20,7 +21,7 @@ export class StoreTriggerCall {
       modelName: payload.metadata.productName,
       flowPayload: payload,
     });
-    const response = await axios.post<StoreTriggerResponse>(`${this.storeTrigger.url}/${this.storeTrigger.subUrl}`, payload);
+    const response = await axios.post<StoreTriggerResponse>(`${this.storeTrigger.url}`, payload);
```

Comment:
If you remove the subUrl, you have to add it hard-coded.. Otherwise, the request will not be sent to the right path.

Reply:
ok